keyboards/hineybush/hbcp/hbcp.c

```diff
@@ -0,0 +1,98 @@
+/* Copyright 2019 Josh Hinnebusch
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "hbcp.h"
+
+// Indicator color definitions
+#define HSV_CAPS  0, 0, 120 // Define caps lock color (H, S, V)
```

Comment:
For users to override these defaults they will be required to `#undef` first, generally we wrap our defaults in a "if not defined then define" check
```suggestion
#ifndef HSV_CAPS
#define HSV_CAPS  0, 0, 120 // Define caps lock color (H, S, V)
#endif
```